cornellh4i/ithaca-recovery
# added zoom account info to meetings

### DESCRIPTION
Updated generateToken.ts to accept an additional parameter to determine which Zoom account (1, 2, 3, or 4) is used for generating access tokens. Updated CreateMeeting/index.ts, DeleteMeeting/index.ts, UpdateMeeting/index.ts to reflext this change.

### TESTING
1. Navigate to http://localhost:3000/test.
2. Test creating, updating, and deleting meetings.
3. Verify functionality with multiple Zoom accounts and check console logs for meeting information

<img width="662" alt="Screenshot 2024-11-23 at 4 58 10 PM" src="https://github.com/user-attachments/assets/c8b99ed2-39df-4ff7-874d-a1577ad0048d">
